.claude/commands/apply-plan.md

```
@@ -0,0 +1,311 @@
+description: Execute an approved plan — per-task TDD, spec + code quality review, plan file cleanup, PR
+argument-hint:
+  [
+    leave empty if a plan was approved in this session,
+    OR openspec:<change-name>,
+    OR PLAN_<slug>.md (explicit plan file path),
+  ]
+allowed-tools: Read, Grep, Glob, Agent, Bash(git:*), Bash(pytest:*), Bash(python:*), Bash(flake8:*), Bash(mypy:*), Bash(black:*), Bash(coverage:*), Bash(rm:*), TodoWrite, mcp__github__*, subscribe_pr_activity
+model: claude-sonnet-4-6
+---
+
+$ARGUMENTS
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)